readme.md

```diff
@@ -12,7 +12,9 @@ Copy both files in `apis/` to `/apis/`, and put `programs/github` wherever you w
 Usage
 -----
 
-    github clone <user>/<repo name> [destination folder]
+    github clone <user>/<repo name> [branch|tag] [destination folder]
+
+The branch or tag argument is optional, and defaults to 'master'.
```

Comment:
That is correct.

On Sun, Aug 21, 2016, 03:04 Eric Wieser <notifications@github.com> wrote:

> In readme.md
> <https://github.com/eric-wieser/computercraft-github/pull/6#discussion_r75590636>
> :
>
> > @@ -12,7 +12,9 @@ Copy both files in `apis/` to `/apis/`, and put `programs/github` wherever you w
> >  Usage
> >  -----
> >
> > -    github clone <user>/<repo name> [destination folder]
> > +    github clone <user>/<repo name> [branch|tag] [destination folder]
> > +
> > +The branch or tag argument is optional, and defaults to 'master'.
>
> I said "commit sha", not object sha. I'm also confused by tree shas vs
> commit shas. My understanding was that a commit object contained a single
> tree, and so tree and commit shas are different things?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eric-wieser/computercraft-github/pull/6/files/f42d18e03d4c4e0664d9cafec698034e170c1a5f#r75590636>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA41FlcB3W60iUNbDtuIy-PmU2HKJ5sPks5qiAaJgaJpZM4JoI_u>
> .
>